src/include/ndpi_typedefs.h

```diff
@@ -536,6 +537,39 @@ struct ndpi_vxlanhdr {
   u_int8_t reserved;
 } PACK_OFF;
 
+
+#define NDPI_GRE_CSUM        ntohs(0x8000)
+#define NDPI_GRE_ROUTING     ntohs(0x4000)
+#define NDPI_GRE_KEY         ntohs(0x2000)
+#define NDPI_GRE_SEQ         ntohs(0x1000)
+#define NDPI_GRE_STRICT      ntohs(0x0800)
+#define NDPI_GRE_REC         ntohs(0x0700)
+#define NDPI_GRE_ACK         ntohs(0x0080)
+#define NDPI_GRE_FLAGS       ntohs(0x00f8)
+#define NDPI_GRE_VERSION     ntohs(0x0007)
+
+#define NDPI_GRE_IS_CSUM(f)		((f) & NDPI_GRE_CSUM)
+#define NDPI_GRE_IS_ROUTING(f)	((f) & NDPI_GRE_ROUTING)
+#define NDPI_GRE_IS_KEY(f)		((f) & NDPI_GRE_KEY)
+#define NDPI_GRE_IS_SEQ(f)		((f) & NDPI_GRE_SEQ)
+#define NDPI_GRE_IS_STRICT(f)	((f) & NDPI_GRE_STRICT)
+#define NDPI_GRE_IS_REC(f)		((f) & NDPI_GRE_REC)
+#define NDPI_GRE_IS_FLAGS(f)		((f) & NDPI_GRE_FLAGS)
+#define NDPI_GRE_IS_ACK(f)		((f) & NDPI_GRE_ACK)
+
+#define NDPI_GRE_VERSION_0 ntohs(0x0000)
+#define NDPI_GRE_VERSION_1 ntohs(0x0001)
```

Comment:
I'd prefer to have 2 direct macros like `NDPI_GRE_IS_VERSION_0(f) ((f) & NDPI_GRE_VERSION) == ntohs(0x0000))` (using them in the code); it would be more coherent with the macros above